src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/visualeditor/VisualEditorPageObject.java

```diff
@@ -208,8 +220,8 @@ public void verifyGalleries(int expected) {
       wait.forElementVisible(galleryNode);
     }
     Assertion.assertNumber(
-            getNumOfElementOnPage(By.cssSelector(".media-gallery-wrapper.ve-ce-branchNode")), expected,
-            "Checking the correct number of gallery nodes");
+        getNumOfElementOnPage(By.cssSelector(".media-gallery-wrapper.ve-ce-branchNode")), expected,
```

Comment:
It isn't my change, honestly. As I can see this function requires argument of type By, and there's no way to give it an argument of WebElement type.